multimodal/tests/hf_model_list.yaml

```diff
@@ -24,6 +24,10 @@ others:
 - sentence-transformers/all-mpnet-base-v2
 - sentence-transformers/msmarco-MiniLM-L-12-v3
 - xlm-roberta-base
+- swin_base_patch4_window7_224
+- yolov3_mobilenetv2_320_300e_coco
+- faster_rcnn_r50_fpn_2x_coco
+- detr_r50_8x2_150e_coco
```

Comment:
This file is only huggingface models. No need to add these models here.

Reply:
oh right. I will delete them.